src/esp/sim/Simulator.cpp

```diff
@@ -217,6 +217,14 @@ Simulator::setSceneInstanceAttributes(const std::string& activeSceneName) {
 
   metadata::attributes::SceneAttributes::cptr curSceneInstanceAttributes =
       metadataMediator_->getSceneAttributesByName(activeSceneName);
+  // check if attributes is null - should not happen
+  CORRADE_ASSERT(
```

Comment:
Is it possible to hit any of these by calling from python into Habitat-sim in a certain way, or by calling into Habitat-sim with certain files on the filesystem missing?

If yes, this feels more like an `ESP_CHECK`.

Reply:
yes, great point, thanks @eundersander